client/src/components/Map.jsx

```diff
@@ -93,6 +144,33 @@ const Map = () => {
         <NavigationControl />
       </div>
     </ReactMapGL>
+    <form
+        className="searchBar"
+        role="search"
+        style={{ position: "absolute", left: "27%", top: 110 }}
+        onSubmit={handleSubmit}
+      >
+        <input
+          class="focus:outline-none focus:shadow-outline pl-2 mr-0.75 rounded-sm h-10 border-2 w-64"
+          type="search"
+          list="suggestions"
+          onChange={onChange}
+          placeholder="Search for a location..."
+          aria-label="Submit"
```

Comment:
You have an aria-label="Submit" here and on line 169 -- I think it might be confusing, so change this line to something else?